core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/ai-content-prompt.component.html

```diff
@@ -0,0 +1,22 @@
+<form
+    id="ai-text-prompt"
+    *ngIf="form"
+    [formGroup]="form"
+    (ngSubmit)="onSubmit()"
+    autocomplete="off"
+>
+    <div>
+        <span class="p-input-icon-right">
+            <input
+                #input
+                formControlName="textPrompt"
+                type="text"
+                pInputText
+                placeholder="Ask AI to write something"
```

Comment:
Needs translation.

Reply:
How do you manage the translation of phrases such as "Ask AI to write something"? I searched but couldn't find any examples.